src/commands/dockersupport/localDockerSupport.ts

```diff
@@ -14,51 +14,43 @@ import { validateDockerInstalled } from "./validateDockerInstalled";
-            if (node.site.reserved) {
-                // asks if the user wants to use Docker for initializing the project locally
-                const useDocker: string = await prompt(context);
-                if (useDocker === "yes") {
-                    // download dev container
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/devcontainer.json`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'devcontainer.json').fsPath
-                    );
-                    // download docker file
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/Dockerfile`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'Dockerfile').fsPath
-                    );
-                    // external - check if Docker is installed, Remote Development extension AND Docker Extension
-                    if (!validateDockerInstalled()) {
-                        // if Docker is not downloaded - ask the user if they'd like to download Docker
-                        const downloadDocker: string = await prompt2(context);
-                        if (downloadDocker === "yes") {
-                            // Check if Operating System is Windows
-                            if (process.platform == "win32") {
-                                // Download Docker with an MSI package
-                            } else {
-                                // Not windows: display link to download docker externally from Docker documentation
-                                void vscode.window.showInformationMessage(localize('downloadDocker', 'Check the (Docker documentation)[https://docs.docker.com/get-docker/] to download Docker for your system'));
-                            }
-                        } else {
-                            void vscode.window.showInformationMessage(localize('noDocker', 'Continuing without the use of Docker as user requested'));
-                        }
+    if (node && devContainerName && node.site.reserved) {
+        // asks if the user wants to use Docker for initializing the project locally
+        const useDocker: string = await prompt(context);
+        if (useDocker === "yes") {
+            await downloadLocalDevFiles(devContainerFolderPathUri, devContainerName);
+            // external - check if Docker is installed, Remote Development extension AND Docker Extension
+            if (!validateDockerInstalled()) {
+                // if Docker is not downloaded - ask the user if they'd like to download Docker
+                if (await prompt2(context) === "yes") {
+                    // Check if Operating System is Windows
+                    if (process.platform == "win32") {
```

Comment:
should use the `===` equality operator.

Reply:
will make this change

Reply:
<img width="605" alt="Screenshot 2021-07-14 182620" src="https://user-images.githubusercontent.com/61916412/125700932-bac56603-1eb9-4320-9483-8fcec9dc7d69.png">
Its not possible to do it with `win64`